PERFORMANCE.md

```diff
@@ -0,0 +1,343 @@
+path = results/inference-colocated-scaling/run-2023-08-17-18:23:38
+smartsim_version = 0.5.0
+smartredis_version = 0.3.1
+db = redis-server
+date = 2023-08-17
+language = ['cpp', 'fortran']
+
+[attributes]
+colocated = 1
+client_per_node = [18]
+client_nodes = [4, 8, 12, 16]
+database_cpus = [8]
+database_port = 6780
+batch_size = [96]
+device = GPU
+num_devices = 1
+iterations = 100
+language = ['cpp', 'fortran']
+node_feature = {'constraint': 'P100'}
+```
+
+#### Put Tensor (send image to database)
+![Inference Colo](/figures/put_tensor_inf_colo.png "Inference Colocated")
+
+#### Run Script (preprocess image)
+![Inference Colo](/figures/run_script_inf_colo.png "Inference Colocated")
+
+#### Run Model (run resnet50 on the image)
+![Inference Colo](/figures/run_model_inf_colo.png "Inference Colocated")
+
+#### Unpack Tensor (retrieve the inference result)
+![Inference Colo](/figures/unpack_tensor_inf_colo.png "Inference Colocated")
+
+## Inference Performance Analysis
+
+In this section, we will compare inference clients: `put-tensor`, `unpack-tensor`, `run_model` and `run_script`
+for colocated and standard deployment.
+
+> Note that Inference is the process of running data points into a machine learning model to calculate an output such as a single numerical score. The SmartSim-Scaling tests use Pytorch's implementation of Resnet50 to feed 
+the ImageNet dataset through. We put the ImageNet data into the database via put_tensor, process the data via run_script, run a machine learning model with the data via run_model then retrieve the data via unpack_tensor.
+
+- `put-tensor` : Colocated deployment offers a consistent median for put_tensor times. Standard deployment shows a slight
+increase in median as client count grows. However, due to machine constraints, colocated is maxed at 288 clients while
+standard maxes at 1800 clients. We can conclude that there is not a significant performance hit putting information into the database when comparing standard and colocated.
+
+- `run_script` : Colocated deployment offers a faster run_script client than standard deployment. We can 
+infer colocated deployment is able to transfer information faster when processing data than standard deployment. 
+This is likely because communication time is cut when using colocated deployment. There are also not as many requests sent using colocated deployment than standard which result in faster completion for colocated.
+
+- `run_model` : Colocated deployment offers a faster run_model client than standard deployment. Like mentioned before,
+the communication time is cut when using colocated since the app and database are on the same node.
+
+- `unpack-tensor` : There is no significant performance advantage when using colocated deployment vs standard for the client
+unpack_tensor. However, standard shows larger outside points than colocated. 
+
+There is no `put-tensor` or `unpack-tensor` performance hit when using standard versus colocated deployment
```

Comment:
Rather than talking about a performance hit using standard, talk about a performance benefit when using colo. Put the positive spin, not the negative one.